ddtrace/span.py

```diff
@@ -484,7 +484,9 @@ def pprint(self):
             ("end", "" if not self.duration else self.start + self.duration),
             ("duration", "%fs" % (self.duration or 0)),
             ("error", self.error),
+            ("sampled", self.sampled),
```

Comment:
this _should_ always be `True`. which would normally make me say omit it, but if it is ever `False` it'll be interesting...

Reply:
ah sure, removed